From 29a48e8f0a2fb7ef935f36a876fadb9ce5e0984a Mon Sep 17 00:00:00 2001
From: Cheng Lian <lian@databricks.com>
Date: Fri, 23 Jan 2015 16:20:10 -0800
Subject: [PATCH 63/93] PARQUET-168: Fixes parquet-tools command line option description

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/apache/incubator-parquet-mr/106)
<!-- Reviewable:end -->

Author: Cheng Lian <lian@databricks.com>

Closes #106 from liancheng/PARQUET-168 and squashes the following commits:

4524f2d [Cheng Lian] Fixes command line option description
---
 .../src/main/java/parquet/tools/Main.java          |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/parquet-tools/src/main/java/parquet/tools/Main.java b/parquet-tools/src/main/java/parquet/tools/Main.java
index e644f3c..a75d968 100644
--- a/parquet-tools/src/main/java/parquet/tools/Main.java
+++ b/parquet-tools/src/main/java/parquet/tools/Main.java
@@ -52,7 +52,7 @@ public class Main {
                                .create();
 
     Option debg = OptionBuilder.withLongOpt("debug")
-                               .withDescription("Disable color output even if supported")
+                               .withDescription("Enable debug output")
                                .create();
 
     OPTIONS.addOption(help);
@@ -95,7 +95,7 @@ public class Main {
     if (usage != null && usage.length >= 1) {
       ustr = ustr + " " + usage[0];
     }
-      
+
     format.printUsage(err, WIDTH, ustr);
     format.printWrapped(err, WIDTH, LEFT_PAD, "where option is one of:");
     format.printOptions(err, WIDTH, options, LEFT_PAD, DESC_PAD);
-- 
1.7.0.4

